### PR TITLE
Updates based on review comments

### PR DESCRIPTION
**Pull request by vivek-arte**
This PR updates the ZSA ZIPs based on the comments made on PR#680.

### CODE REVIEW

File: zip-0227.rst [hunk 1]
```diff
@@ -205,16 +210,18 @@ The relations between the Asset Identifier, Asset Digest, and Asset Base are sho
...
+**Note:** To keep notations light and concise, we may omit :math:`\mathsf{AssetId}` in the subscript when the Asset Identifier is clear from the context.
 
 Wallets MUST NOT display just the :math:`\mathsf{asset\_desc}` string to their users as the name of the Asset. Some possible alternatives include:
 
 - Wallets could allow clients to provide an additional configuration file that stores a one-to-one mapping of names to Asset Identifiers via a petname system. This allows clients to rename the Assets in a way they find useful. Default versions of this file with well-known Assets listed can be made available online as a starting point for clients.
 - The Asset Digest could be used as a more compact bytestring to uniquely determine an Asset, and wallets could support clients scanning QR codes to load Asset information into their wallets.
 
+If the :math:`\mathsf{asset\_desc}` string is not a well-formed UTF-8 code unit sequence, the wallet SHOULD use the byte string as is in their method for displaying the Asset.
```
Comment (vivek-arte) [inline zip-0227.rst:223]: TODO: Add words saying "the wallet should display a warning" to that effect

File: zip-0227.rst [hunk 2]
```diff
@@ -337,13 +312,18 @@ If all of the above checks pass, do the following:
 - If :math:`\mathsf{finalize} = 1`, add :math:`\mathsf{AssetDigest}` to the :math:`\mathsf{previously\_finalized}` set immediately after the block in which this transaction occurs.
 - (Replay Protection) If issue bundle is present, the fees MUST be greater than zero.
 
+Rationale for non-zero Issue Bundle fees
```
Comment (PaulLaux) [inline zip-0227.rst:315]: Rationale for non-zero fees when an issue bundle is included.

File: zip-0227.rst [hunk 3]
```diff
@@ -337,13 +312,18 @@ If all of the above checks pass, do the following:
...
+----------------------------------------
+We require non-zero fees in the presence of an issue bundle, in order to preclude the possibility of a transaction containing only an issue bundle.
```
Comment (PaulLaux) [inline zip-0227.rst:317]: "in order to preclude the possibility of a replay attack."

Then, describe the potential attack.

File: zip-0227.rst [hunk 4]
```diff
@@ -205,15 +210,18 @@ The relations between the Asset Identifier, Asset Digest, and Asset Base are sho
...
+**Note:** To keep notations light and concise, we may omit :math:`\mathsf{AssetId}` in the subscript when the Asset Identifier is clear from the context.
 
-Wallets MUST NOT display just the :math:`\mathsf{asset\_desc}` string to their users as the name of the Asset. Some possible alternatives include:
+The Asset Base is the unique identifier of the Asset in the Orchard-ZSA Protocol.
+Wallets MUST NOT display just the :math:`\mathsf{asset\_desc}` string to their users as the name of the Asset.
+If the :math:`\mathsf{asset\_desc}` string is not a well-formed UTF-8 code unit sequence, the wallet MUST provide a warning that the Asset description cannot be parsed as a UTF-8 encoded string, and SHOULD use the byte string as is in their method for displaying the Asset description.
+
+Wallets could allow clients to provide an additional configuration file that stores a one-to-one mapping of names to Asset Bases via a petname system. This allows clients to rename the Assets in a way they find useful. Default versions of this file with well-known Assets listed can be made available online as a starting point for clients.
```
Comment (PaulLaux) [inline zip-0227.rst:222]: avoid the term `petname system`. use auxiliary naming system.

File: zip-0227.rst [hunk 5]
```diff
@@ -205,15 +210,18 @@ The relations between the Asset Identifier, Asset Digest, and Asset Base are sho
...
+The Asset Digest could be used as a more compact bytestring to identify an Asset, and wallets could support clients scanning QR codes to load Asset information into their wallets.
```
Comment (PaulLaux) [inline zip-0227.rst:223]: Asset base

Comment (PaulLaux) [reply zip-0227.rst:223]: This entire sentence feels redundant.

**Review by PaulLaux — APPROVED** [review]
Added some minor comments.

File: zip-0227.rst (hunk 2 above)
Comment (vivek-arte) [reply zip-0227.rst:315]: Done

File: zip-0227.rst (hunk 3 above)
Comment (vivek-arte) [reply zip-0227.rst:317]: Done

File: zip-0227.rst (hunk 4 above)
Comment (vivek-arte) [reply zip-0227.rst:222]: Done

File: zip-0227.rst (hunk 5 above)
Comment (vivek-arte) [reply zip-0227.rst:223]: True, removed it.

**Pull request merged**